plugins/MauticCitrixBundle/Entity/CitrixEvent.php

```diff
@@ -1,262 +0,0 @@
-<?php
-
-namespace MauticPlugin\MauticCitrixBundle\Entity;
-
-use Doctrine\ORM\Mapping as ORM;
-use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
-use Mautic\LeadBundle\Entity\Lead;
-
-#[ORM\Table(name: 'plugin_citrix_events')]
-#[ORM\Entity]
-class CitrixEvent
-{
-    #[ORM\Column(name: 'id', type: \Doctrine\DBAL\Types\Types::INTEGER)]
-    #[ORM\Id]
-    #[ORM\GeneratedValue]
-    protected ?int $id = null;
-
-    /**
-     * @var Lead
-     */
-    protected $lead;
-
-    #[ORM\Column(name: 'product', type: \Doctrine\DBAL\Types\Types::STRING, length: 20)]
-    protected ?string $product = null;
-
-    #[ORM\Column(name: 'email', type: \Doctrine\DBAL\Types\Types::STRING, length: 255)]
-    protected ?string $email = null;
-
-    #[ORM\Column(name: 'event_name', type: \Doctrine\DBAL\Types\Types::STRING, length: 255)]
-    protected ?string $eventName = null;
-
-    #[ORM\Column(name: 'event_desc', type: \Doctrine\DBAL\Types\Types::STRING, length: 255)]
-    protected ?string $eventDesc = null;
-
-    #[ORM\Column(name: 'event_type', type: \Doctrine\DBAL\Types\Types::STRING, length: 50)]
-    protected ?string $eventType = null;
-
-    #[ORM\Column(name: 'event_date', type: \Doctrine\DBAL\Types\Types::DATETIME_MUTABLE)]
-    protected ?\DateTimeInterface $eventDate = null;
-
-    public function __construct()
-    {
-        $this->product   = 'undefined';
-        $this->email     = 'undefined';
-        $this->eventName = 'undefined';
-        $this->eventDesc = 'undefined';
-        $this->eventDate = new \Datetime();
-        $this->eventType = 'undefined';
-    }
-
-    public static function loadMetadata(ORM\ClassMetadata $metadata)
-    {
-        $builder = new ClassMetadataBuilder($metadata);
-        $builder->setTable('plugin_citrix_events')
```

Comment:
We'll also need a migration to drop this table. The replacement plugin does not use this table. It creates a different one for events: https://github.com/Leuchtfeuer/mautic-goto-bundle/blob/staging/Entity/GoToEvent.php#L64

Reply:
True, we should mention it in the release notes. In case they'd want to somehow migrate from the core plugin to the replacement. But once the code itself is removed then it's a dead table. Nothing will read nor write from it.

Reply:
@escopecz I added a migration to remove table in 0a9670f0a8f8827098b4cd2605fa7a52b1ab0b65